impeller/entity/entity_pass_target.cc

```diff
@@ -45,7 +45,16 @@ std::shared_ptr<Texture> EntityPassTarget::Flip(Allocator& allocator) {
     }
   }
 
-  std::swap(color0.resolve_texture, secondary_color_texture_);
+  // If the color0 resolve texture is the same as the texture, then we're
+  // running on the GLES backend with implicit resolve.
+  if (color0.resolve_texture == color0.texture) {
```

Comment:
aside: I wish we could do a capabilities check instead, but this is better than nothing I suppose

Reply:
If you want me to I can pass that in? I'll just do that.

Reply:
Done

Reply:
Up to you. I can't imagine another scenario but at least we wouldn't accidentally capture a bug versus an intent. Thanks!